web/docs/operators/timeshift.md

```diff
@@ -0,0 +1,64 @@
+---
+sidebar_custom_props:
+  operator:
+    transformation: true
+---
+
+# timeshift
+
+Adjusts timestamps relative to a given start time, with an optional speedup.
+
+## Synopsis
+
+```
+timeshift [--start <time>] [--speed <factor>] <field>
+```
+
+## Description
+
+The `timeshift` operator adjusts a series of time values by anchoring them
+around a given start time.
+
+If you do not provide a start time with `--start`, the operator will anchor the
+timestamps in `field` to begin with the current wall clock time, as if you
+provided `--start now`.
```

Comment:
As I explained in the comment on the thread, this behavior is incompatible with only applying a speedup. Omitting `--start` should behave just like for `delay`, where the start time is then the first non-null time value encountered.

Reply:
Is it correctly described in 8a0a805194?